doc/stdlib/parallel.rst

```diff
@@ -299,8 +299,6 @@ General Parallel Computing Support
 
    Called by cluster managers using custom transports. It should be called when the custom transport implementation receives the first message from a remote worker. The custom transport must manage a logical connection to the remote worker and provide two ``IO`` objects, one for incoming messages and the other for messages addressed to the remote worker. If ``incoming`` is ``true``\ , the remote peer initiated the connection. Whichever of the pair initiates the connection sends the cluster cookie and its Julia version number to perform the authentication handshake.
 
-.. function:: Future()
```

Comment:
Eaten by a rebase, I think.
> On Jul 31, 2016, at 19:17, Tony Kelman <notifications@github.com> wrote:
> 
> In doc/stdlib/parallel.rst <https://github.com/JuliaLang/julia/pull/17681#discussion_r72917653>:
> 
> > @@ -299,8 +299,6 @@ General Parallel Computing Support
> >  
> >     Called by cluster managers using custom transports. It should be called when the custom transport implementation receives the first message from a remote worker. The custom transport must manage a logical connection to the remote worker and provide two ``IO`` objects, one for incoming messages and the other for messages addressed to the remote worker. If ``incoming`` is ``true``\ , the remote peer initiated the connection. Whichever of the pair initiates the connection sends the cluster cookie and its Julia version number to perform the authentication handshake.
> >  
> > -.. function:: Future()
> where did this go?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/JuliaLang/julia/pull/17681/files/c5307064cef6a7b31f0ea8c3e4e8c69d5fe9b240..ed2f97f71ed7b876a6989284cffefb8250baeaa2#r72917653>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAyk47Iek7ZNt8vYmH8GesblknEEWny1ks5qbVdUgaJpZM4JXmvy>.
>